### ISSUE TITLE
How can i extract coefficients for each variable from a BigVAR object?

### ISSUE THREAD

**Issue by runnytone** [wbnicholson/BigVAR#10]
Thanks for your contribution.
How can I extract coefficients for each variable from a BigVAR object?
Thank you.

**Comment by runnytone** [comment]
I find that BigVAR.results returns the array of VAR/VARX coefficients from out of sample forecasts. But I was told that  `could not find function "BigVAR.results"` every time I call the function.

**Comment by runnytone** [comment]
I tried the demo below. Does '@betaPred' call the coeffcients for the bigvar?
`
library(BigVAR)
# VARX Example
# Create a Basic VARX-L with k=2, m=1, s=2, p=4
VARX=list()
VARX$k=2 # indicates that the first two series are modeled
VARX$s=2 # sets 2 as the maximal lag order for exogenous series
data(Y)
T1=floor(nrow(Y)/3)
T2=floor(2*nrow(Y)/3)
Model1=constructModel(Y,p=4,struct="Basic",gran=c(50,10),verbose=FALSE,VARX=VARX,T1=T1,T2=T2)
est <- cv.BigVAR(Model1)
B <- est@betaPred
`

**Comment by wbnicholson** [comment]
Yes, betaPred returns the coefficients for the last model fit in the evaluation period.
